samples/one-liners.md

```diff
@@ -180,3 +179,13 @@ pcon ${opts} | awk -vjob="${job}" '
  /^O|^M|^N|^W|^X/ { if ($6 == job) { trace=1; print; } else { trace=0; }  }
  /^S|^L|^E|^D/  { if (trace) { print; } }
 ```
+
+**Data Set Operations**
+- Delete data set `HLQ.MLQ.LLQ` using mvscmd and idcams  
+  `echo " DELETE HLQ.MLQ.LLQ " |mvscmdauth --verbose --pgm=IDCAMS --sysprint=* --sysin=stdin`
+
+- Delete data set `HLQ.MLQ.LLQ` on a particular volume using mvscmd and idcams  
+  `echo "  DELETE HLQ.MLQ.LLQ FILE(DD1) NVR" | mvscmdauth --pgm=IDCAMS --sysprint=* --dd1=USERVOL,vol --sysin=stdin`
+
+- Rename data set `HLQ.MLQ.LLQ` to `USER.MLQ.LLQ`using mvscmd and idcams  
```

Comment:
Same reason above, was just demonstrating how one uses mvscmdauth with a volume. I spent some time helping someone assemble this command as they were struggling as the command in the tool they were using was not obvious how it was assembled and then whether it was correct and they wanted to trouble shoot so I helped them and figured if it had been here it would have saved us both time.

Reply:
I can unpack a tar.gz with one command `tar -xf archive.tar.gz` or i can use two commands `tar` and `gunzip` point being there is always more than one way to do things, why not let users see how to do those things in case they actually can't use a preferred way.

Reply:
Makes sense. Can you add a reference to the Z Open Automation Utilities command too and point out the catalog point?